build/tasks/compare_size.mjs

```diff
@@ -22,6 +37,17 @@ async function getCache( loc ) {
 	}
 }
 
+function cacheResults( results ) {
+	const files = {};
```

Comment:
It’s a bit safer to do:
```suggestion
	const files = Object.create( null );
```
to avoid weird behavior with a possible `__proto__` branch.